lib/web_of_science/process_records.rb

```diff
@@ -61,22 +60,11 @@ def select_new_wos_records
       def save_wos_records
         return if records.empty?
         process_links
-        records.select! do |rec|
```

Comment:
This `select!` never did any selecting, by definition, since `.save!` was never going to return false, it would just explode!